playgrounds/angular/tsconfig.json

```diff
@@ -8,14 +8,29 @@
     "declaration": false,
     "downlevelIteration": true,
     "experimentalDecorators": true,
+    "allowSyntheticDefaultImports": true,
     "moduleResolution": "node",
     "importHelpers": true,
     "target": "es2015",
     "module": "es2020",
     "lib": [
       "es2020",
       "dom"
-    ]
+    ],
+    // Enforced typing rules
+    "strict": true,
+    "noImplicitAny": true,
+    "strictNullChecks": true,
+    "strictFunctionTypes": true,
+    "strictPropertyInitialization": true,
+    "strictBindCallApply": true,
+    "noImplicitThis": true,
+    "noImplicitReturns": true,
+    "alwaysStrict": true,
+    "emitDecoratorMetadata": true,
+    "forceConsistentCasingInFileNames": true,
+    "noUnusedLocals": false,
+    "noUnusedParameters": true,
```

Comment:
To verify. Also according to the documentation if you use `"strict": true,` it should enabled all the strict mode familly:
- noImplicitAny
- strictNullChecks (since 2.0)
- noImplicitThis (since 2.0)
- strictFunctionTypes (since 2.6)
- strictPropertyInitialization (since 2.7)
- strictBindCallApply (since 3.2)

So if you use it you don't need the others.